packages/nextjs/src/config/loaders/wrappingLoader.ts

```diff
@@ -202,8 +202,13 @@ export default function wrappingLoader(
       templateCode = templateCode.replace(/__COMPONENT_TYPE__/g, 'Unknown');
     }
 
-    if (sentryConfigFilePath) {
-      templateCode = `import "${sentryConfigFilePath}";`.concat(templateCode);
+    if (sentryConfigFilePath && path.isAbsolute(this.resourcePath)) {
+      const sentryConfigFileImportPath = path
+        .relative(this.resourcePath, sentryConfigFilePath) // Get path relative to current module because webpack can't handle absolute paths on windows: https://github.com/getsentry/sentry-javascript/issues/8133
```

Comment:
This needs to be `path.dirname(this.resourcePath)` otherwise the relative path goes up one level too far

Reply:
Yea that seems to be the most important thing!